### PR TITLE
aws aod support

### PR DESCRIPTION
**Pull request by qili86**
Tested, Verified, shared the screenshots with @arpitjasa-db and @eric-golinko-db

### CODE REVIEW

File: template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl [hunk 1]
```diff
@@ -144,12 +144,18 @@ By default, we provide Azure Pipelines where authentication is done using servic
 to get your service principal credentials (tenant id, application id, and client secret) for both the staging and prod service principals. You will use these credentials as variables in the project Azure Pipelines.
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl:144]: We'll need to change these lines as well: https://github.com/databricks/mlops-stacks/pull/114/files#diff-af86930ef42587a2d047c0da274d89a5401de2c1cabef648cae6893cf1dcd8d1R142-R143 and maybe other parts below. For example with the lines mentioned, we want to change `Azure Databricks` to just `Databricks` and `Follow ['Get Azure AD tokens...` to `Get PAT tokens` etc

Comment (qili86) [reply template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl]: Yep, as I am using the it to set up the CICD on ADO, I will keep updating this

**Comment by arpitjasa-db** [comment]
@qili86 is this ready for review?

**Comment by qili86** [comment]
> @qili86 is this ready for review?

@arpitjasa-db Yes!

**Review by arpitjasa-db — APPROVED** [review]
LGTM thanks @qili86 !

**Pull request merged**